src/sentry/integrations/gitlab/integration.py

```diff
@@ -28,11 +30,39 @@
 )
 
 
+API_ERRORS = {
+    404: 'Gitlab returned a 404 Not Found error. If this repository exists, ensure'
+         ' that your installation has permission to access this repository',
+    401: ERR_UNAUTHORIZED,
+}
+
+
 class GitlabIntegration(IntegrationInstallation):
 
     def get_client(self):
         pass
 
+    def reinstall(self):
```

Comment:
Im not sure this will work, the group id should be the same, but I'm not sure what the reinstall flow will look like here. We also don't have any disabling in place yet, and not sure if we will?